Makefile

```diff
@@ -216,38 +214,15 @@ pyclean:
 
 # Build docker image
 docker docker-cuda:
-	docker build -t $(DOCKER_TAG) --build-arg BASE_IMAGE=$(DOCKER_BASE_IMAGE) $(DOCKER_ARGS) .
+	docker build --progress=plain -t $(DOCKER_TAG) --build-arg BASE_IMAGE=$(DOCKER_BASE_IMAGE) $(DOCKER_ARGS) .
```

Comment:
:+1: will make the build log more readable.

Reply:
with the newer progress meter, there simply is no build log (the output simply gets overwritten by new output along the way)